### ISSUE TITLE
about policies

### ISSUE THREAD

**Issue by limbo127** [percolate/iamer#3]
hello, 
i'm using iamer 0.1.4, with access_key with full iam right, 
but policies does not seem to be dump.
user.ini and group.ini is ok , no policies in it, and no policies in policies directory
regards

**Comment by lra** [comment]
That's right, `iamer` does not support managed policies, only inline ones.
Not specific reason, it's just a feature that was not there when we built `iamer`.

Feel free to do a PR, or just wait and we'll eventually add it in ;)

**Comment by lra** [comment]
Keeping this open until it's done.
